src/test/api.spec.ts

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import axios from "axios";
+// `require`, don't `import` moxios because the type definition is currently broken.
+const moxios = require("moxios");
```

Comment:
See the comment on line 17. The DefinitelyTyped file for moxios is incomplete and using it leads to (incorrect) TypeErrors in our test suite.